generators/app/templates/src/layouts/PrivateLayout/private.layout.js

```diff
@@ -18,35 +20,58 @@ const Content = styled.div`
   overflow-y: auto;
 `;
 
-const PrivateLayout = ({ routes, webId, location, history, ...rest }) => (
-  <React.Fragment>
-    <Container>
-      <Route
-        {...rest}
-        component={({ history }) => (
-          <Content className="contentApp">
-            <AuthNavBar {...{ location, webId, history }} />
-            <Switch>
-              {routes.map(route => {
-                const { component: RouteComponent } = route;
-                return (
-                  <Route
-                    key={route.id}
-                    path={route.path}
-                    render={routerProps => <RouteComponent {...routerProps} webId={webId} />}
-                    webId={webId}
-                    exact
-                  />
-                );
-              })}
-              <Redirect to="/404" />
-            </Switch>
-          </Content>
-        )}
-      />
-      <Footer />
-    </Container>
-  </React.Fragment>
-);
+const PrivateLayout = ({ routes, webId, location, history, ...rest }) => {
+  const { t } = useTranslation();
+  const checkAppPermissions = useCallback(async () => {
+    const permission = await AppPermission.checkPermissions(webId);
+    /**
+     * SDK app will need all the permission by the user pod,
+     * so for this reason we check it for that.
+     */
+    const permissions = permission.permissions.toString();
+
+    if (permissions !== process.env.REACT_APP_PERMISSIONS) {
+      errorToaster(t('appPermission.message'), 'Error', {
```

Comment:
"Error" should be coming from i18n, even though it's the same in english and spanish. The translation has already been added, so it's just a matter of using the translation

Reply:
done